shared/js/sk_key_utils.js

```diff
@@ -0,0 +1,328 @@
+(function () {
+
+  'use strict';
+
+  // Keys which are mapped to accesskey
+  const KEYS_MAPPING = {
+    LSK: 'F1',
+    CSK: 'Enter',
+    RSK: 'F2'
+  };
+
+  // Storing the reference to keys
+  // which will show options menu
+  var elements = {
+    LSK: null,
+    CSK: null,
+    RSK: null
+  };
+
+  // Container created for SoftKey
+  var container = null;
+
+  // Check if menu open
+  var options_menu = {
+    open: false,
+    cancelKey: KEYS_MAPPING.RSK
+  };
+
+  // Observer for observing the DOM changes
+  var observer = null;
+
+  //////////////////////////////
+
+  // Initializing the module. Will be called once only.
+  document.addEventListener(
+          'DOMContentLoaded',
+          init);
+
+  //////////////////////////////
+
+  /**
+   * Used to initialize the library
+   * @example
+   * init();
+   *
+   * @param N/A
+   * @return void 0
+   */
+  function init() {
+
+    console.log("Init called for library");
+    // Container will be created to handle
+    // soft keys
+    // A #IFDEF check is required for
+    // NON-TOUCH UI
+    createSoftKeyContainer();
+    
+    var optionsMenu = document.getElementById('sample-menu');
+    if(optionsMenu){
+      document.getElementById('sample-menu').style.display = 'none';
```

Comment:
Rather than using the hidden attribute here, it is probably better to set and remove a CSS class.